# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/Preferences.cpp
# Commit: 0abcd848f821
# Full Hash: 0abcd848f8218f0e2d339bbd497ac60f4a3b752c
# Author: Tom Ritter <tom@mozilla.com>
# Date: 2023-06-30 21:10:55
# Regressor Bug: 1840851
# File Overlap Count: 1
# Description:
#   Bug 1840851: Add a few prefs to the allowlist r=freddyb
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D182359
# ==============================================================================

diff -r 444443efbf7a -r 0abcd848f821 modules/libpref/Preferences.cpp
--- a/modules/libpref/Preferences.cpp	Fri Jun 30 12:04:25 2023 +0000
+++ b/modules/libpref/Preferences.cpp	Fri Jun 30 13:48:54 2023 +0000
@@ -6123,6 +6123,7 @@
     PREF_LIST_ENTRY("logging.config.LOG_FILE"),
     PREF_LIST_ENTRY("media.audio_loopback_dev"),
     PREF_LIST_ENTRY("media.decoder-doctor."),
+    PREF_LIST_ENTRY("media.cubeb.backend"),
     PREF_LIST_ENTRY("media.cubeb.output_device"),
     PREF_LIST_ENTRY("media.getusermedia.fake-camera-name"),
     PREF_LIST_ENTRY("media.hls.server.url"),
@@ -6137,10 +6138,12 @@
     PREF_LIST_ENTRY("network.security.ports.banned"),
     PREF_LIST_ENTRY("nimbus.syncdatastore."),
     PREF_LIST_ENTRY("pdfjs."),
+    PREF_LIST_ENTRY("plugins.force.wmode"),
     PREF_LIST_ENTRY("print.printer_"),
     PREF_LIST_ENTRY("print_printer"),
     PREF_LIST_ENTRY("places.interactions.customBlocklist"),
     PREF_LIST_ENTRY("remote.log.level"),
+    PREF_LIST_ENTRY("services.settings.loglevel"),
     PREF_LIST_ENTRY(
         "services.settings.preview_enabled"),  // This is really a boolean
                                                // dynamic pref, but one Nightly
